build/media-suite_compile.sh

```diff
@@ -2432,27 +2438,107 @@ if [[ $mpv != n ]] && pc_exists libavcodec libavformat libswscale libavfilter; t
             done
         fi
 
+        # Translate the Waf flags to Meson options instead.
+        # It should be fairly easy to overwrite the mpv_options file,
+        # though differentiating would require ugly hacks like a header marker.
+        # This allows reusing old configs and doesn't rely on correctly formatted input,
+        # since users editing a config may easily slip up with `-D` prepended to every line.
+        # The alternative is having users only specify everything after it in the config, e.g.
+        # `lua=luajit` and `cdda=enabled` (optionally omitting `=enabled` for on/off features)
+        local meson_opts=()
+        local default_lib="both"
+        local option
+        for option in "${MPV_OPTS[@]}"; do
+            # Process Waf flags into Meson options.
+            # starting with boolean flags
+            if [[ $option =~ ^--enable-(gpl|cplayer|build-date|tests|ta-leak-report) ]]; then
```

Comment:
TA Leak can only be turned off by also adjusting an env var; perhaps make some accessible entry point for people to inject the option from a custom patch instead of setting it here?